src/main/java/com/example/decafe/Customer.java

```diff
@@ -109,99 +96,53 @@ public static void setControllerTimer(Timer controllerTimer) { //sets the timer
 
     // Method used to create an Image Object
```

Comment:
Remove this function and make the customer class using the single responsibility method in ImageUtil : ImageUtil.getImageFromResources(filename)